10_PredictionExamples.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<blockquote>Since we will be using xs_train as data, we are left with: </blockquote><p>I have no idea what this means. Values to predict?</p><p> </p><p>I would change the header to "Prediction" and be more explicit in the text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>